### ISSUE TITLE
Install on Pi 3

### ISSUE THREAD

**Issue by TwinkleToes423** [rethinkdb/rethinkdb_rebirth#57]
I am attempting to install RethinkDB on a Raspberry Pi 3b. I follow your build instructions, however, it never makes it past process 19. I have tried a fresh install of the OS, I also attempted both the versions for the Pi and Debian. Noting seems to work. What do you recommend?

**Comment by MurugappanV** [comment]
Are you not using SWAP partition? SWAP partition of atleast 1GB is required, by default only 100MB is available in Pi3

**Comment by TwinkleToes23** [comment]
Yes I am, i dug deeper and there is a version mismatch on the supporting libraries.

**Comment by MurugappanV** [comment]
> Yes I am, i dug deeper and there is a version mismatch on the supporting libraries.

how did you solved the version mismatch?

**Comment by TwinkleToes23** [comment]
It isnt something I can solve. The devs need to handle it.
